runtime/testy/studentAPI.py

```diff
@@ -74,11 +74,31 @@ def get_value(self, name, gamepad_number=0):
       raise StudentAPIKeyError()
 
 class Robot(StudentAPI):
+  deviceType_to_validParams = {
+    10 : ["duty_cycle", "enable"], 
+  }
+  param_to_valid_values = {
+    "duty_cycle" : ["float", -1, 1],
+    "enable" : ["boolean"],
+  }
+
   def __init__(self, toManager, fromManager):
     super().__init__(toManager, fromManager)
     self._createSensorMapping()
     self._coroutines_running = set()
 
+  def get_value(self, device_name, param):
+    uid = self._hibikeGetUID(device_name)
```

Comment:
Pretty sure this should through an error if the key doesn't exist, and thus can not ever be `None`